src/pages/ProductCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import { Transition } from "@headlessui/react";
+import Image from "next/image";
+
+export default function ProductCard({
+  content,
+  index,
+}: {
+  content: any;
```

Comment:
Do your best not to have a type of any, that's very dangerous and defeates the purpose of typescript

Reply:
added type object and passed to content: Content.